src/core/modules/Asterism.cpp

```diff
@@ -75,13 +75,21 @@ bool Asterism::read(const QString& record, StarMgr *starMgr)
 			case 0: // Ray helpers
 			case 1: // A big asterism with lines by HIP stars
 			{
-				unsigned int HP = 0;
+				int64_t HP = 0;
 				istr >> HP;
 				if(HP == 0)
 				{
 					return false;
 				}
-				asterism[i]=starMgr->searchHP(static_cast<int>(HP));
+				if (HP <= 120416)
```

Comment:
The same change has to go into Constellation::read().
Maybe this if.. clause should already go into a method StarMgr::searchStar(int64_t starIdx), and the 120416 can be kept inside StarMgr.

Reply:
On a separate note, this number must not be magical, but have a name.